README.md

```diff
@@ -83,8 +83,8 @@ Change into the directory with ORT's source code and run `docker build -t ort .`
 
 Install these additional prerequisites:
 
-* OpenJDK 8 or Oracle JDK 8u161 or later (not the JRE as you need the `javac` compiler); also remember to set the
-  `JAVA_HOME` environment variable accordingly.
+* Java Development Kit (JDK) version 11 or later (not the JRE as you need the `javac` compiler); also remember to set
```

Comment:
For Java 11 no JRE standalone is available anymore. Therefore you could drop the text in the parentheses.
See: https://www.heise.de/download/product/java-runtime-environment-jre-627 and https://www.heise.de/download/product/java-runtime-environment-jre-627 (german)

Reply:
Done.